rotkehlchen/chain/evm/decoding/stakedao/decoder.py

```diff
@@ -74,17 +74,17 @@ def reload_data(self) -> Mapping['ChecksumEvmAddress', tuple[Any, ...]] | None:
         """
         if should_update_protocol_cache(
             cache_key=CacheType.STAKEDAO_GAUGES,
-            args=(str(self.evm_inquirer.chain_id.value),),
+            args=(str(self.evm_inquirer.chain_id.serialize()),),
         ) is True:
-            query_stakedao_gauges()
+            query_stakedao_gauges(self.evm_inquirer)
```

Comment:
don't you need to clear the cache or something so it can be requeried now again?